app/config.cpp

```diff
@@ -89,7 +89,7 @@ Config LoadConfig(std::string path)
 
     std::ifstream in{};
     in.open(path, std::ios::in);
-    auto data = nlohmann::json::parse(in);
+    auto data = nlohmann::json::parse(in, nullptr, true, true);
```

Comment:
Can we change this to something like:
```
const  bool allow_comments = true;
const  bool other_true_variable = true;
... parse(in, nullptr, allow_comments, other_true_variable);
```
it's not clear to me just from looking at the function call what the other true variable is and which goes where.

Reply:
Yes. That will be clearer.

I set 2nd and 3rd parameter to their default values. 4th parameter (ignore_comments) is the only one I wanted to explicitly set to true (default false).